misk-testing/src/main/kotlin/misk/service/ServiceTestingModule.kt

```diff
@@ -15,63 +13,26 @@ import kotlin.reflect.KClass
  * that is being spun up within the test itself
  */
 class ServiceTestingModule<T : Service> internal constructor(
-  private val wrappedServiceKey: Key<T>,
+  private val wrappedServiceClass: KClass<T>,
```

Comment:
I don't understand why it has to be by Class and not Key?

Reply:
Services are bound using multi binder, but there are binding strategies

```
multibind<Service>().to<FooService>()
multibind<Service>().toProvider(FooServiceProvider(...))
multibind<Service>().to(Key.get(FooService)
multibind<Service>().toInstance(FooService(...))
```

If the binding strategy is provider or instance, then there's no key for the linked binding. The only way I can think of to target a specific service is to check the instance's class.